mingw-w64-highlight/PKGBUILD

```
@@ -0,0 +1,27 @@
+_realname=highlight
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=4.2
+pkgrel=1
+url="http://www.andre-simon.de/doku/highlight/highlight.html"
+pkgdesc="Fast and flexible source code highlighter (CLI version)"
+license=('GPL')
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
+             "${MINGW_PACKAGE_PREFIX}-lua"
+             "${MINGW_PACKAGE_PREFIX}-boost")
+_commit='80c5735e1f2a525c4f85e5f220c6b3c2bb26c063'
+source=("https://gitlab.com/saalen/highlight/-/archive/${_commit}/highlight-${_commit}.tar.gz")
```

As this is a tagged release, I think a specific tarball can be used. Please test the following

```suggestion
source=("https://gitlab.com/saalen/highlight/-/archive/v${pkgver}/highlight-v${pkgver}.tar.bz2")
```

---

@Biswa96 Tagged release doesn't build on MSYS2.  Upstream hasn't made a release yet since my upstream fixes.  Another option to still go with a released version (as opposed to a commit id) would be have the same fixes a patches downstream.  Do you recommend that?

---

You can use the tagged source tarball and patch the source code with the upstream or your patches.

---

@Biswa96 Done!  I've used version 4.3 which contains all my upstream fixes :)